src/main/java/TaskList.java

```diff
@@ -0,0 +1,93 @@
+import java.util.ArrayList;
+
+/**
+ * TaskList class is a specific class that contains the task storage, a
+ * static variable to collect user inputs, and manipulates it accordingly.
+ */
+public class TaskList {
+    /**
+     * The arraylist to store all user input tasks.
+     */
+    public static ArrayList<Task> taskStorage = new ArrayList<>();
```

Comment:
Should the variable name have a plural form (e.g. "tasks") since the variable represents a collection of objects?